packages/itwinui-css/src/button/base.scss

```diff
@@ -41,7 +30,17 @@
   svg {
     @include mixins.iui-icon-style('m');
     transition: fill var(--iui-duration-1) ease-out;
-    fill: var(--_iui-button-icon-fill, currentColor);
+    fill: var(--_iui-button-icon-fill, var(--iui-color-icon));
```

Comment:
I believe we had decided that it would be safer to have the button text and the button icon match in color, and so we had preferred `currentColor` (https://github.com/iTwin/iTwinUI/pull/1787/files/33c0c084ad43990e3038576bb0002246cd70b17a#r1459075467).

Reply:
I still feel `currentColor` would be safer.

Just curious, is there a strong reason/benefit to use `--iui-color-icon`? If so, and if we're confident that we're setting the `--_iui-button-icon-fill` variable correctly, then `--iui-color-icon` may be good too.

Reply:
We discussed this in a call and have decided to stick with `--iui-color-icon` rather than switch it to `currentColor`.